pandas/tests/reshape/test_reshape.py

```diff
@@ -302,6 +302,26 @@ def test_dataframe_dummies_with_categorical(self, df, sparse, dtype):
         expected.sort_index(axis=1)
         assert_frame_equal(result, expected)
 
+    def test_dataframe_dummies_unicode(self):
+        df = pd.DataFrame(({u'ä': ['a']}))
+        result = get_dummies(df)
+        expected = pd.DataFrame({u'ä_a': [1]}, dtype=np.uint8)
+        assert_frame_equal(result, expected)
+
+        df = pd.DataFrame({'x': [u'ä']})
+        result = pd.get_dummies(df)
+        expected = pd.DataFrame({u'x_ä': [1]}, dtype=np.uint8)
+        assert_frame_equal(result, expected)
```

Comment:
This one would pass even without a fix, but I've included it for completeness.

Reply:
More coverage is never a problem for us 🙂